parser.y

```diff
@@ -6886,20 +6886,33 @@ ShowStmt:
                         User:	$4.(*auth.UserIdentity),
                 }
         }
-|	"SHOW" "TABLE" TableName "REGIONS"
+|	"SHOW" "TABLE" TableName "REGIONS" WhereClauseOptional
 	{
-		$$ = &ast.ShowStmt{
+
+		stmt := &ast.ShowStmt{
 			Tp:	ast.ShowRegions,
 			Table:	$3.(*ast.TableName),
 		}
+		if $5 != nil {
+			stmt.Where = $5.(ast.ExprNode)
+		}
+		$$ = stmt
 	}
-|	"SHOW" "TABLE" TableName "INDEX" Identifier "REGIONS"
+|	"SHOW" "TABLE" TableName "INDEX" Identifier "REGIONS" ShowLikeOrWhereOpt
```

Comment:
Should this still be `ShowLikeOrWhereOpt`?

Reply:
Done. Thanks.